server/sonar-server/src/main/java/org/sonar/server/computation/task/projectanalysis/issue/ComponentIssuesLoader.java

```diff
@@ -41,7 +46,7 @@ public ComponentIssuesLoader(DbClient dbClient, RuleRepository ruleRepository, A
     this.ruleRepository = ruleRepository;
   }
 
-  public List<DefaultIssue> loadForComponentUuid(String componentUuid) {
+  public List<DefaultIssue> loadForComponentUuid(String componentUuid, boolean loadChanges) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Provide multiple methods instead of using "loadChanges" to determine which action to take. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS2301)